docs/testing.md

```diff
@@ -8,64 +8,64 @@ Unit tests are useful for simple standalone functionality, and CRUD operations.
 
 [Mocked external keepers](./testutil/keeper/mocks.go) (implemented with [gomock](https://github.com/golang/mock)) are available for testing code that briefly interacts with external modules, but still only a single function/method relevant to ccv, and a single chain. Ie. do not use mocked external keepers to test the integration of the ccv module with external modules, or integration between consumer and provider.
 
-## End to End (e2e) Tests
+## Integration Tests
```

Comment:
When this PR is up to date with main, the link to this file will be broken in the top level readme. Might be worth fixing in this PR